scripts/presubmit.sh

```diff
@@ -112,9 +112,18 @@ main() {
   if [[ "${run_lint}" -eq 1 ]]; then
     check_cmd golangci-lint \
       'have you installed github.com/golangci/golangci-lint?' || exit 1
+    if ! type -p "prototool" > /dev/null; then
+      TMP="$(mktemp -d)"
```

Comment:
Probably better to use a more specific name (or possibly a `local` name) in case there's a `$TMP` elsewhere.

Reply:
still to do